src/NoteMouseHandler/PencilMouseHandler.js

```diff
@@ -1,17 +1,18 @@
 import _ from "lodash"
 import SharedService from "../services/SharedService"
-import MouseHandler, { defaultActionFactory, getLocal } from "./NoteMouseHandler"
+import MouseHandler, { defaultActionFactory } from "./NoteMouseHandler"
 import { pointSub } from "../helpers/point"
 
+const defaultCursor = `url("./images/iconmonstr-pencil-14-16.png") 0 16, default`
+
 export default class PencilMouseHandler extends MouseHandler {
-  constructor(emitter) {
-    super(emitter, [defaultActionFactory, actionFactory], getCursor)
+  constructor() {
+    super([defaultActionFactory, actionFactory], getCursor, defaultCursor)
   }
 }
 
-function actionFactory(e, ctx) {
-  const cpos = getLocal(e)
-  const rects = ctx.getEventsUnderPoint(cpos.x, cpos.y)
+function actionFactory(local, ctx, e) {
+  const rects = ctx.getEventsUnderPoint(local.x, local.y)
```

Comment:
__[jshint]__
*  'const' is available in ES6 (use 'esversion

<!-- Comment by [SideCI](https://sideci.com) -->